server/tests/helpers/index.ts

```diff
@@ -0,0 +1 @@
+import './core-utils'
```

Comment:
Add this test to `travis.sh` (in `misc` for example)

Reply:
Ok done, let's cross our fingers that it fixed the misc test